plugins/woocommerce/client/admin/client/marketplace/components/quality-badge/quality-badge.scss

```
@@ -0,0 +1,105 @@
+@import "../../stylesheets/_variables.scss";
```

Stylelint (17.14.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)